### PR TITLE
XIVY-12449 windows integration tests

### PR DESCRIPTION
**Pull request by ivy-lmu**

### CODE REVIEW

File: build/integration/windows/Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,53 @@
+pipeline {
+  agent {
+    label { label 'windows-build01 || windows-build02 || windows-build03' }
```
Comment (ivy-lmu) [inline build/integration/windows/Jenkinsfile:3]: not working with 04 - black screen, no idea why

Comment (ivy-lli) [reply build/integration/windows/Jenkinsfile:3]: maybe other setup or someone changed something. or logged in per rcp?

**Review by ivy-lli — APPROVED** [review]
Noice

File: build/integration/windows/Jenkinsfile (hunk 1 above)
Comment (ivy-lmu) [reply build/integration/windows/Jenkinsfile:3]: I will restart 04 and double check it

Comment (ivy-lmu) [reply build/integration/windows/Jenkinsfile:3]: restart did not solve it, I leave that as it is for now ;)

**Pull request merged**